dask/array/core.py

```diff
@@ -222,7 +222,11 @@ def slices_from_chunks(chunks):
     return list(product(*slices))
 
 
-def getem(
```

Comment:
We should check with Xarray tests to make sure that we don't break anything there.  They may use this (I hope not though).